src/harness/templates/NOTES.txt

```diff
@@ -0,0 +1,10 @@
+# Chart 0.2.87
+
+- All module level overrides must be present at root level in override file and NOT under `harness` key
```

Comment:
we can add something like:
previously:

harness:
  srm:
     enable-receivers: true